### PR TITLE
Adding MSigDB Permutations

### PR DESCRIPTION
**Pull request by gwaygenomics**
Script modified from @dhimmel 's original at https://github.com/dhimmel/integrate/blob/master/permute.ipynb

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,3 +2,8 @@ hetnets/msigdb_nodes.tsv filter=lfs diff=lfs merge=lfs -text
 hetnets/msigdb_hetnet.json.bz2 filter=lfs diff=lfs merge=lfs -text
 hetnets/msigdb_edges.sif.gz filter=lfs diff=lfs merge=lfs -text
 results/msigdb_node_degrees.xlsx filter=lfs diff=lfs merge=lfs -text
+hetnets/permuted/msigdb_hetnet_perm-1.json.bz2 filter=lfs diff=lfs merge=lfs -text
+hetnets/permuted/msigdb_hetnet_perm-2.json.bz2 filter=lfs diff=lfs merge=lfs -text
+hetnets/permuted/msigdb_hetnet_perm-3.json.bz2 filter=lfs diff=lfs merge=lfs -text
+hetnets/permuted/msigdb_hetnet_perm-4.json.bz2 filter=lfs diff=lfs merge=lfs -text
+hetnets/permuted/msigdb_hetnet_perm-5.json.bz2 filter=lfs diff=lfs merge=lfs -text
```
Comment (dhimmel) [inline .gitattributes]: I'll usually just do *.bz2 because I know that there will never be a .bz2 file that I don't want to LFS track

**Comment by dhimmel** [comment]
From your commit message "track all bz2 files", I think you may have a misunderstanding. The following line in `.gitattributes':

```
*.bz2 filter=lfs diff=lfs merge=lfs -text
```

This does not track all bz2 files. It just specifies when tracking a .bz2 file, to track with LFS. Files in .gitignore that end in .bz2 will continue not to be tracked, and all .bz2 files still need to be added.

**Comment by gwaygenomics** [comment]
> if you don't plan to run evaluate-permutations.ipynb, then I have no outstanding comments.

I plan on running today - would be good to check and also to increase understanding of what the permutations are doing

**Comment by gwaygenomics** [comment]
Looks like `xswap` is working well - probably can lower `multiplier` to 3

**Comment by dhimmel** [comment]
> probably can lower multiplier to 3

Yep looks like you could get away with 3, but so leeway is not a bad thing... unless you'd like this to run faster. I'd say stick with 5 for now since you've already run it.

**Pull request merged**